features/profiles/profiles.api.ts

```diff
@@ -12,86 +11,78 @@ const ProfileSchema = z.object({
   updatedAt: z.string(),
 });
 
-export type IProfile = z.infer<typeof ProfileSchema>;
+export type IConvosProfileForInbox = z.infer<
+  typeof ConvosProfileForInboxSchema
+>;
 
-const SearchProfilesResultSchema = z.object({
-  id: z.string(),
-  name: z.string(),
-  description: z.string().nullable(),
-  xmtpId: z.string(),
-});
-
-export type ISearchProfilesResult = z.infer<typeof SearchProfilesResultSchema>;
+export const updateProfile = async ({
+  id,
+  updates,
+}: {
+  id: string;
+  updates: { name?: string; description?: string };
+}) => {
+  const { data } = await api.put(`/api/v1/profiles/${id}`, updates);
+  return ConvosProfileForInboxSchema.parse(data);
+};
 
-// todo get profile needs to use xmtpId i believe
-export const getProfile = async (id: string): Promise<IProfile> => {
-  const { data } = await api.get(`/api/v1/profiles/${id}`);
-  return ProfileSchema.parse(data);
+export const fetchProfile = async ({
+  xmtpId,
+}: {
+  xmtpId: string;
+}): Promise<IConvosProfileForInbox> => {
+  const { data } = await api.get(`/api/v1/profiles/${xmtpId}`);
+  return ConvosProfileForInboxSchema.parse(data);
 };
 
-export const getAllProfilesForUser = async ({
+export const fetchAllProfilesForUser = async ({
   convosUserId,
 }: {
   convosUserId: string;
-}): Promise<IProfile[]> => {
+}): Promise<IConvosProfileForInbox[]> => {
   const { data } = await api.get(`/api/v1/profiles/user/${convosUserId}`);
```

Comment:
Usually the best practices route would be more like `/api/v1/user/${convosUserId}/profiles`

Reply:
just gonna follow what's there for now we can change later